dashboard/app/models/levels/level.rb

```diff
@@ -206,11 +206,18 @@ def available_callouts(script_level)
   # Output: Hash of level properties
   def load_level_xml(xml_node)
     hash = JSON.parse(xml_node.xpath('//../config').first.text)
-    if hash['encrypted_properties']
-      hash['properties'] = Encryption.decrypt_object(hash.delete('encrypted_properties'))
-    end
-    if hash['encrypted_notes']
-      hash['notes'] = Encryption.decrypt_object(hash.delete('encrypted_notes'))
+    begin
+      encrypted_properties = hash.delete('encrypted_properties')
```

Comment:
Presumably this refactoring is a cleanup and not strictly required?  It looks good, just making sure I'm not missing a functional change..

Reply:
without refactoring, the `encrypted_notes` property was not removed, and I got an error complaining about that. so I had to rearrange it to remove both encrypted fields before doing any steps which could fail and be rescued from.